reana_server/reana_admin.py

```diff
@@ -589,11 +592,14 @@ def set_quota_limit(ctx, admin_access_token, emails, resource_name, limit):
 def set_default_quota_limit(ctx, admin_access_token):
     """Set default quota limits to users that do not have any."""
     users_without_quota_limits = User.query.filter(~User.resources.any()).all()
+    if not users_without_quota_limits:
+        click.secho("There are no users without quota limits.", fg="green")
+        sys.exit(1)
```

Comment:
Question: I guess it was failing because the function wasn't idempotent?

Reply:
Not really, when I tested it locally there were no users without quota limits, so `users_without_quota_limits = []` and this was causing some errors while invoking `set_quota_limit`